doc/source/examining/loading_data.rst

```diff
@@ -1926,6 +1926,63 @@ available here are similar to other catalogs.  Any addition
    # The halo mass
    print(ad["halos", "particle_mass"])
 
+.. _ytree_data:
+
+ytree Merger Trees
+------------------
+
+`ytree <https://ytree.readthedocs.io/>`_ is an extension of yt for working
+with merger tree data from multiple sources. Merger trees loaded with ytree
+can be re-written to a format that can be loadable by both ytree and yt.
+Loading merger tree data in yt allows one to access halos using yt's geometric
+data containers instead of from within the tree structures used by ytree. For
+example, this can be useful if one wants to quickly access all halos in the
+merger tree at once.
+
+The ytree data format consists of a directory containing a series of HDF5 files.
+To load the dataset, provide the path to the HDF5 file with the same name as
+the directory.
+
+.. code-block:: python
+
+   import yt
+   ds = yt.load("arbor/arbor.h5")
+   # all halo masses
+   print (ds.r['particle_mass'])
```

Comment:
I thought `ds.r` was for selecting a region in coordinate space.  But I guess you can use it to just select all the values of a given field for a dataset like (`ad = ds.all_data(), ad['field']`)?  I guess I'm just confused because I didn't know this was implemented this way...Can someone confirm that this is used elsewhere in yt?

Reply:
Looks like I'm just behind the times on this.  OK, everything LGTM.